docs/prysm-usage/grafana-dashboard.md

```diff
@@ -0,0 +1,145 @@
+---
+id: grafana-dashboard
+title: Monitoring and alerting with Grafana
+sidebar_label: Monitoring with Grafana
+---
+
+Now that you have your validator and node process running, you will probably need a nice dashboard and alert system to ensure at maximum the profitability of your staking ETH. Here is a simple guide that explain how to get one, without any developer skill.
+
+Here is the result of what you will get if you follow this guide!
+![Grafana dashboard for prysm node and validator](https://github.com/prysmaticlabs/documentation/blob/master/docs/prysm-usage/grafana-dashboards/assets/dashboard_overview.png?raw=true "Grafana dashboard for prysm node and validator")
```

Comment:
All image files need to go under `documentation/website/static/img/`, these are within the docs repo

Reply:
You can't see them? You mean while viewing the .md file? This is because I have preset the url that will have the image once it will be merge to master on this repository. So for now, the image doesn't exist (since they are not yet on this repository)

I'll change the images right now

Reply:
thanks! The file paths can also be changed to `/img/<filename>`, that'll work when they are live